stdpopsim/genetic_maps.py

```diff
@@ -53,8 +50,25 @@ def __init__(
 
     @property
     def map_cache_dir(self):
+        """
+        The path to the directory in which the files for this map are stored.
+        Names for individual chromosome files within this directory can be obtained
+        using :attr:`file_pattern`.
+
+        :type: pathlib.PosixPath
```

Comment:
The type here actually depends on the OS. On Unix it's `PosixPath`, but on Windows it will presumably be `WindowsPath`. So probably map_cache_dir's type should be listed as `pathlib.Path` ([the super type of both](https://docs.python.org/3/library/pathlib.html#concrete-paths)).